conbench/api/results.py

```diff
@@ -290,8 +285,6 @@ def post(self) -> f.Response:
         # errors that are to be exposed via a 400 response to the HTTP client
         benchmark_result = BenchmarkResult.create(data)
 
-        set_z_scores([benchmark_result])
```

Comment:
Makes a lot of sense to me that this is gone! (same for all PUT/POST code paths here in this PR).